src/Calendar/Calendar.js

```diff
@@ -118,8 +188,7 @@ class Calendar extends Component {
     changeMonth = (month) => {
         const newDate = moment(this.state.currentDateDisplayed)
             .locale(this.props.locale)
-            .month(month)
-            .date(1);
+            .month(month);
```

Comment:
careful: This was *probably* put in place so that you don't accidentally jump from 1/31/2020 to 2/31/2020

Reply:
With moment, it will now set it to the latest day if the new month doesn't have enough days. Can go either way on this though.